CHANGELOG.rst

```diff
@@ -6,12 +6,12 @@ Rasa Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to `Semantic Versioning`_ starting with version 1.0.
 
-[Unreleased 1.2.3] - `master`_
+[Unreleased 1.3] - `master`_
```

Comment:
You should create a new section for the release (eg by moving the items from the collected master section) and create a new master logging section